src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -725,6 +725,13 @@ def load_arguments(self, _):
             c.argument('identity_role', options_list=['--role'], arg_group=arg_group, help="Role name or id the system assigned identity will have")
             c.ignore('identity_role_id')
 
+    with self.argument_context('vm auto-shutdown') as c:
+        c.argument('disable', arg_type=get_three_state_flag(), help='Disable auto-shutdown for VM')
```

Comment:
Why use `disable` rather than `enable` ?

Reply:
Use `--disable` to disable it.
If no `--disable`, it will enable.

Reply:
I give it a fantastic name `--off`